entrypoint.sh

```diff
@@ -12,17 +12,32 @@ if [ -z "$INPUT_CONTAINER_NAME" ]; then
   exit 1
 fi
 
-if [ -z "$INPUT_CONNECTION_STRING" ]; then
-  echo "storage account connection string is not set. Quitting."
+CONNECTION_METHOD=""
+
+if ! [ -z "$INPUT_CONNECTION_STRING" ]; then
+  CONNECTION_METHOD="--connection-string $INPUT_CONNECTION_STRING"
+elif ! [ -z "$INPUT_SAS_TOKEN" ]; then
+  if ! [ -z "$INPUT_ACCOUNT_NAME" ]; then
+    CONNECTION_METHOD="--sas-token $INPUT_SAS_TOKEN --account-name $INPUT_ACCOUNT_NAME"
+  else
+    echo "account_name is required if using a sas_token. account_name is not set. Quitting."
+    exit 1
+  fi
+else
+  echo "either connection_string or sas_token are required and neither is set. Quitting."
   exit 1
 fi
 
-EXTRA_ARGS=${INPUT_EXTRA_ARGS:""}
+EXTRA_ARGS=""
+if [ -z "$INPUT_EXTRA_ARGS" ]; then
```

Comment:
@wilsonnm02 this should be `if ! [ -z "$INPUT_EXTRA_ARGS" ]; then` otherwise it will always ignore `INPUT_EXTRA_ARGS` 🙏